customTests/azure_gpu_ecc.nhc

```diff
@@ -98,6 +98,7 @@ function check_SDBE_ecc(){
    collect_ecc_data "SDBE"
    flag="false"
    error_msg=''
+   re='^[0-9]+$'
```

Comment:
is this a bug fix?

Reply:
yes to accommodate for N/A output that can appear in the nvsmi retirement table query